README.md

```diff
@@ -12,13 +12,13 @@ The contents of this code of conduct apply to our interactions in various areas
 
 ## Values
 
-We want our team to be a fun, productive, and safe space for all members. In addition to core values of respect and honesty, there are several ways in which our product team distinguishes itself.
+We want our team to be fun, productive, and safe for all members. In addition to the Social Tables [core values](https://www.socialtables.com/company/about/#values), there are several ways in which our Engineering team distinguishes itself.
```

Comment:
I think it would make sense to do that, though it isn't necessary. An Engineering-only CoC certainly makes sense, and a company-wide CoC also makes sense too (though that would require a much higher level of coordination w/ Staff than we might want to bite off right now). Involving Design and/or Product might be a happy medium, or even a second step.